data/json/scenarios.json

```diff
@@ -311,8 +311,7 @@
       "RAP_TALONS",
```

Comment:
```suggestion
```
Adds additional attack

Reply:
I think fangs should also be removed on 304.  But I can't touch it currently because you didn't change anything near it.